8.8/jhbuildrc

```diff
@@ -34,12 +34,6 @@ addpath('PKG_CONFIG_PATH', os.path.join(prefix, 'lib', 'pkgconfig'))
 # or it'll see host .pc files
 os.environ['PKG_CONFIG_LIBDIR'] = os.path.join(prefix, 'lib', 'pkgconfig')
 
-# .pc files from binary packages have something like 
-#     prefix=c:/devel/target/f3002dbeeb43fd4ea5385fb6fa4de06f
-# at the top ... we need to override this prefix with our own
-# don't use "--define-variable=prefix=/poop", some pkg-config get confused
-os.environ['PKG_CONFIG'] = '/usr/bin/pkg-config --define-variable prefix=' + prefix
```

Comment:
Removed, otherwise the build of poppler fails:
https://cgit.freedesktop.org/poppler/poppler/tree/CMakeLists.txt?id=248540fbe44886f7ede290c216822df21ad06d7d#n679